Pickers/SWActionSheet.m

```diff
@@ -9,7 +9,7 @@
 
 static const float duration = .25f;
 
-static const enum UIViewAnimationOptions options = UIViewAnimationOptionCurveEaseIn;
+static const enum UIViewAnimationOptions options = 7 << 16;
```

Comment:
Where it comes from this magic constant? Can you present it as combination of `UIViewAnimationOptions` ?